routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
+
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"success": true,
+		"data":    todoList,
+	})
+
+}
+
+func (h handler) GetTodo(c *fiber.Ctx) error {
```

Comment:
Validation here is insufficient, on a large number (which is of a valid type) the API returns as if it wasn't.
Since you are not logging the actual problem you have no way of knowing this error exists or how you can fix it.

<img width="1500" alt="Screenshot 2023-07-08 at 14 13 22" src="https://github.com/kevinpita/go-todo/assets/84069271/94a71243-8b72-4714-8ed9-0ad8b38e1df4">

Reply:
Once again, validation here is non-existent, a comma is not a valid number, and shouldn't return 200 when the logic doesn't know we are not trying to retrieve all the todos:

<img width="1509" alt="Screenshot 2023-07-08 at 14 14 55" src="https://github.com/kevinpita/go-todo/assets/84069271/c71f21d0-dd80-45cc-b189-9823eb27bbdf">

Reply:
> Validation here is insufficient, on a large number (which is of a valid type) the API returns as if it wasn't. Since you are not logging the actual problem you have no way of knowing this error exists or how you can fix it.
> <img alt="Screenshot 2023-07-08 at 14 13 22" width="1500" src="https://user-images.githubusercontent.com/84069271/251971779-94a71243-8b72-4714-8ed9-0ad8b38e1df4.png">

Fixed logging 6459f3dcbfa22b99a2a904e9290e4c14b39acf7b and improved message 5fb2763b0bde2afe5948b1c6c45fddb1d2a3bf84

Reply:
> Once again, validation here is non-existent, a comma is not a valid number, and shouldn't return 200 when the logic doesn't know we are not trying to retrieve all the todos:
> <img alt="Screenshot 2023-07-08 at 14 14 55" width="1509" src="https://user-images.githubusercontent.com/84069271/251971831-c71f21d0-dd80-45cc-b189-9823eb27bbdf.png">

fixed a24bfa701752e650fa687874c0f44ec07016468b

Reply:
> Once again, validation here is non-existent, a comma is not a valid number, and shouldn't return 200 when the logic doesn't know we are not trying to retrieve all the todos:
> <img alt="Screenshot 2023-07-08 at 14 14 55" width="1509" src="https://user-images.githubusercontent.com/84069271/251971831-c71f21d0-dd80-45cc-b189-9823eb27bbdf.png">


I cannot replicate that issue 
![error](https://github.com/kevinpita/go-todo/assets/69063372/6cff913b-d38c-48a3-aa63-75241e1facd2)

Reply:
Me neither, some other part of your code fixed it